cypress/e2e/finance/funds/test-acquisition-unit-restrictions-for-funds.cy.js

```diff
@@ -0,0 +1,98 @@
+import permissions from '../../../support/dictionary/permissions';
+import testType from '../../../support/dictionary/testTypes';
+import devTeams from '../../../support/dictionary/devTeams';
+import FiscalYears from '../../../support/fragments/finance/fiscalYears/fiscalYears';
+import TopMenu from '../../../support/fragments/topMenu';
+import Ledgers from '../../../support/fragments/finance/ledgers/ledgers';
+import Users from '../../../support/fragments/users/users';
+import Funds from '../../../support/fragments/finance/funds/funds';
+import FinanceHelp from '../../../support/fragments/finance/financeHelper';
+import SettingsMenu from '../../../support/fragments/settingsMenu';
+import AcquisitionUnits from '../../../support/fragments/settings/acquisitionUnits/acquisitionUnits';
+
+describe('ui-finance: Transactions', () => {
```

Comment:
the test fails when running on the cypress env

Reply:
![image](https://user-images.githubusercontent.com/17912474/195805521-65d4e681-3d6f-4366-87d4-abf553c1c9c5.png)

Reply:
Please create a ticket for this issue